s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/query/QueryBuilderImpl.java

```diff
@@ -5,14 +5,17 @@
 import java.util.List;
 import java.util.Map;
 
-public class QueryBuilderImpl implements QueryBuilder{
+public class QueryBuilderImpl implements QueryBuilder {
     @Override
     public void prepareQueryAndParameters(Map<String, Object> params,
                                           List<Object> parameters,
                                           StringBuilder resultQuery) {
         resultQuery.append("SELECT " +
                 Scene.COLUMN_ID + "," +
                 Scene.COLUMN_PRODUCT_ID + "," +
-                Scene.COLUMN_TIMESTAMP+ "  FROM Scene WHERE true ");
+                "ST_AsText(" + Scene.COLUMN_FOOTPRINT + ") AS footprint," +
+                Scene.COLUMN_METADATA + "," +
+                Scene.COLUMN_CONTENT + "," +
+                Scene.COLUMN_TIMESTAMP + "  FROM Scene WHERE true ");
```

Comment:
1. The separation of columns will increase the readability of the query.
2. `+/-/*` or `/` signs should always be at the beginning of the line (helps reading code)
3. Instead of adding strings, you should use some strings formatting that places values inside you're a chain of chars. Example from JS `${variable}`

```suggestion
    static String[] COLUMNS = [Scene.COLUMN_ID, ...];
    static String COLUMNS_JOIN = COLUMNS.join(',');
    resultQuery.append(
      `SELECT ${COLUMNS_JOIN},  "ST_AsText(${Scene.COLUMN_FOOTPRINT}) AS footprint`
      + ' FROM Scene'
      + ' WHERE true '
    );
```